test/blockchain/log-processors/crowdsourcers.js

```diff
@@ -23,6 +23,19 @@ const getDisputesFromCrowdsourcer = (db, params, callback) => {
   db("disputes").where({crowdsourcerId: params.log.disputeCrowdsourcer}).asCallback(callback);
 };
 
+const getCrowdsourcerAndMarket  = (db, params, callback) => {
+  getCrowdsourcer(db, params, (err, crowdsourcer) => {
+    if (err) return callback(err);
+    db("markets").first(["marketId", "reportingRoundsCompleted"])
```

Comment:
Since it sounds like you might expand this to query reportingState, I have a helper called `getMarketsWithReportingState` that pre-joins the tables (and includes some defaults)